packages/core/src/server/graphql/buildGraphqlSchema.ts

```diff
@@ -15,15 +14,7 @@ import { buildSingularField } from "./singular.js";
 
 // TODO(kyle) stricter type
 export type Parent = Record<string, any>;
-export type Context = HonoContext<{
-  Variables: { store: IndexingStore; getLoader: GetLoader };
-}> & {
-  get: {
-    <key extends "store" | "getLoader">(_: key): key extends "store"
-      ? IndexingStore
-      : GetLoader;
-  };
-};
+export type Context = { store: IndexingStore; getLoader: GetLoader };
```

Comment:
Do you know why hono wants the context to be accessed with get/set? Might be possible that they are doing some splitting or static analysis under the hood.

Reply:
With this change, we're drawing a distinction between the HTTP request context (managed by e.g. Hono, express) and the GraphQL context. I'm not sure why Hono uses get/set.